src/Experimental/Edit.NET/EditDotNET.csproj

```diff
@@ -4,7 +4,7 @@
         <OutputType>Exe</OutputType>
         <Nullable>enable</Nullable>
         <PackAsTool>true</PackAsTool>
-        <ToolCommandName>Edit.NET</ToolCommandName>
+        <ToolCommandName>editdotnet</ToolCommandName>
```

Comment:
Can't we keep it to be edit.net or something? Or is it used as a package name?

Reply:
yeah, this probably shouldn't be in this delta, we can't have the package name be edit.net (because you have to be abel to publish it) but the**tool name ** can be edit.net

Reply:
sure